src/Symfony/Component/Intl/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+5.4
```

Comment:
you should target 6.4 as new feature :)

Reply:
I mean mostly the target branch of this PR :)

Reply:
Surely it can be merged into older versions as there are no BC breaks? Is it a new feature or a bug fix if it's completing the missing ISO 3166-1 support for numeric codes? I don't mind either way just wanted to clarify...

Reply:
A PR title that starts with "Add support for…" hardly passes as a bugfix. 😉 Let's target 6.4.